include/drivers/pcie/endpoint/pcie_ep.h

```diff
@@ -0,0 +1,157 @@
+/**
+ * @file
+ *
+ * @brief Public APIs for the PCIe EP drivers.
+ */
+
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright 2020 Broadcom
+ *
+ */
+#ifndef ZEPHYR_INCLUDE_DRIVERS_PCIE_EP_H_
+#define ZEPHYR_INCLUDE_DRIVERS_PCIE_EP_H_
+
+#include <device.h>
+#include <init.h>
+#include <kernel.h>
+#include <stdint.h>
+
+enum pcie_ob_mem_type {
+	PCIE_OB_ANYMEM,
+	PCIE_OB_LOWMEM,		/* Within 32-bit address range */
+	PCIE_OB_HIGHMEM,	/* Above 32-bit address range */
+};
+
+enum pci_ep_irq_type {
+	PCIE_EP_IRQ_LEGACY,
+	PCIE_EP_IRQ_MSI,
+	PCIE_EP_IRQ_MSIX,
+};
+
+struct pcie_ep_driver_api {
+	u32_t (*conf_read)(struct device *dev, unsigned int offset);
+	void (*conf_write)(struct device *dev, unsigned int offset,
+			   unsigned int data);
+	int (*map_addr)(struct device *dev, u64_t pcie_addr,
+			u64_t *mapped_addr, unsigned int size,
+			enum pcie_ob_mem_type ob_mem_type);
+	void (*unmap_addr)(struct device *dev, u64_t mapped_addr);
+	int (*raise_irq)(struct device *dev, enum pci_ep_irq_type irq_type,
+			 unsigned int irq_num);
+};
+
+/*
+ * @brief       Read PCIe EP configuration space
```

Comment:
just one space after the doxygen keyword

Reply:
ACK